calculus/source/02-DF/01.ptx

```diff
@@ -425,7 +425,7 @@ Defined and continuous on the interval <m>[-5,5]</m>.
                         </li>
                         <li>
                             <p>
-<m>\displaystyle\lim_{h\to 0} \frac{f( 2 +h)-f( 2 )}{h}&lt; 0</m>
+<m>\displaystyle\lim_{h\to 0} \dfrac{f( 2 +h)-f( 2 )}{h}&lt; 0</m>
```

Comment:
Another potential enhancement, rather than littering `\displaystyle`s for limits, we could try something like https://tex.stackexchange.com/questions/161073/always-using-displaystyle-only-for-lim